src/command_modules/azure-cli-feedback/azure/cli/command_modules/feedback/custom.py

```diff
@@ -555,7 +559,18 @@ def _get_parent_proc_name():
     return _ISSUES_TEMPLATE_PREFIX.format(pretty_url_name), formatted_issues_url, original_issue_body
 
 
-def _get_minified_issue_url(command_log_file, format_dict, is_ext, capacity):
+def  _get_extension_repo_url(ext_name, raw=False):
+    try:
+        project_url = resolve_project_url_from_index(extension_name=ext_name)
+        if 'https://github.com' in project_url:
+            return project_url.strip('/') + _EXTENSION_REPOS_URL_TO_ISSUES_URL
+    except BaseException as ex:
```

Comment:
Why catch such a broad exception? Please catch CLI error and any other exceptions you deem necessary